CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# v2.1.0 (2024-07-08)
+
+## OS Changes
+* Update kernels: 5.10.218-208, 5.10.219, 5.15.160-104, and 6.1.94 ([#13], [#17])
```

Comment:
We should only include the resulting version, we can skip intermediates that never "made it" like 5.10.218

Reply:
Ack, have removed `5.10.218-208`